core/client/app/components/gh-uploader.js

```diff
@@ -28,14 +28,14 @@ var PostImageUploader = Ember.Component.extend({
         $this.on('imagecleared', function () {
             self.sendAction('canceled');
         });
-    }.on('didInsertElement'),
+    },
 
-    removeListeners: function () {
+    willDestroyElement: function () {
```

Comment:
It shouldn't be `willDestroyElement: function removeListeners() {...}` because then you're making the statement that the *only* thing going on in that hook is the removal of listeners.  When something else goes in, is the method going to be renamed `removeListenersAndThisOtherThing()`?

For things that aren't straightforward and obvious inline comments can be used.